packages/components/src/components/hds/form/super-select/single/field.hbs

```diff
@@ -12,8 +11,7 @@
   as |F|
 >
   {{! Notice: the order of the elements is not relevant here, because is controlled at "Hds::Form::Field" component level }}
-  {{yield (hash Label=F.Label isRequired=F.isRequired isOptional=F.isOptional)}}
-  {{yield (hash HelperText=F.HelperText Error=F.Error)}}
+  {{yield (hash Label=F.Label HelperText=F.HelperText Error=F.Error)}}
```

Comment:
Do you not need to pass isRequired/isOptional anymore?

Reply:
[We recently discovered that these were not needed](https://github.com/hashicorp/design-system/pull/2240/files#diff-9b906f806254a94975bab30d6ba8378b8d41572aff5eac578645ec7911bfff04R7), as they are part of the label/legend API